src/navigation-bar.tsx

```diff
@@ -0,0 +1,563 @@
+}));
+
+const BurgerMenuIcon = ({isOpen}: {isOpen: boolean}) => {
+    const classes = useBurgerStyles();
+    return (
+        <div className={classes.burgerIconContainer} role="presentation">
+            <div className={isOpen ? '' : classes.iconCloseHidden}>
+                <IconCloseRegular />
+            </div>
+            <div className={isOpen ? classes.iconMenuHidden : ''}>
+                <IconMenuRegular />
+            </div>
+        </div>
+    );
+};
+
+const NAVBAR_ZINDEX = 25;
+const BURGER_ZINDEX = 26;
+
+const DESKTOP_NAVBAR_HEIGHT = 80;
+const MOBILE_NAVBAR_HEIGHT = 56;
+
+const BURGER_MENU_ANIMATION_DURATION_MS = 300;
+
+const useStyles = createUseStyles((theme) => {
+    const shadowAlpha = theme.isDarkMode ? 1 : 0.2;
+
+    return {
+        topFixed: {
+            position: 'fixed',
+            top: 0,
+            left: 0,
+            right: 0,
+            zIndex: NAVBAR_ZINDEX,
+        },
+        navbar: {
+            width: '100%',
+            display: 'flex',
+            alignItems: 'center',
+            background: ({isInverse}) =>
+                isInverse ? theme.colors.navigationBarBackground : theme.colors.background,
+            height: DESKTOP_NAVBAR_HEIGHT,
+            padding: '16px 0',
+            transition: 'border 300ms',
+            borderBottom: `1px solid ${theme.colors.divider}`,
+            borderColor: ({isInverse}) =>
+                isInverse && !theme.isDarkMode ? 'transparent' : theme.colors.divider,
+            [theme.mq.tabletOrSmaller]: {
+                borderColor: ({isMenuOpen, isInverse}) =>
+                    isMenuOpen || (isInverse && !theme.isDarkMode) ? 'transparent' : theme.colors.divider,
+                height: MOBILE_NAVBAR_HEIGHT,
+                padding: '8px 0',
+            },
+        },
+        section: {
+            height: DESKTOP_NAVBAR_HEIGHT,
+            display: 'flex',
+            alignItems: 'center',
+            padding: '0 8px',
+            borderBottom: `2px solid transparent`,
+            transition: 'border-color 300ms ease-in-out',
+
+            // Only apply hover effect to user agents using fine pointer devices (a mouse, for example)
+            // Also enabled for (pointer: none) for acceptance tests, where (pointer: fine) doesn't match.
+            // WARNING: you may be tempted to use @media (hover: hover) instead, but that doesn't work as expected in some android browsers.
+            // See: https://hover-pointer-media-query.glitch.me/ and https://github.com/mui-org/material-ui/issues/15736
+            '@media (pointer: fine), (pointer: none)': {
+                '&:hover span': {
+                    color: ({isInverse}) =>
+                        isInverse ? theme.colors.textSecondaryInverse : theme.colors.textSecondary,
+                },
+            },
+        },
+        selectedSection: {
+            borderColor: ({isInverse}) => (isInverse ? theme.colors.inverse : theme.colors.controlActivated),
```

Comment:
should we mark the selected section also in mobile/tablet devices (I think it is not being marked in the menu when it is open)

Reply:
Not according design specs. Already discussed with design team